test/Tereshchenko_Igor/laboratory1.sql

```diff
@@ -1,2 +1,158 @@
--- LABORATORY WORK 1
--- BY Tereshchenko_Igor
+
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+виконувати вибірку даних та створювати і змінювати таблиці. 4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+-- USER SQL
+CREATE USER tereshchenko IDENTIFIED BY tereshchenko 
+DEFAULT TABLESPACE "USERS"
+TEMPORARY TABLESPACE "TEMP";
+
+-- QUOTAS
+ALTER USER tereshchenko QUOTA 100M ON USERS;
+
+-- ROLES
+GRANT "CONNECT" TO tereshchenko ;
+
+-- SYSTEM PRIVILEGES
+GRANT CREATE TABLE TO tereshchenko ;
```

Comment:
`TABLE TO tereshchenko` =>`ANY TABLE TO tereshchenko`

Reply:
Fixed